models/movie.model.js

```diff
@@ -0,0 +1,127 @@
+const filename =
+  process.env.HOME + "/Desktop/Programming/Ontotext/MoviesAPI/data/movies.json";
+let movies = require(filename);
+const helper = require("../helpers/helper.js");
+
+/**
+ * This function returns all the movies
+ *
+ * @returns {Promise} Returns all the movies
+ */
+function getMovies() {
+  return new Promise((resolve, reject) => {
+    if (movies.length === 0) {
+      reject({
+        message: "No movies available!",
+        status: 202,
+      });
+    }
+    resolve(movies);
+  });
+}
+
+/**
+ * This function returns a movie by given id
+ *
+ * @param {number} id The movie id
+ * @returns {Promise} Returns the movie if present
+ */
+function getMovie(id) {
```

Comment:
I'm little confused here. The helper.mustBeInArray returns a Promise, right? Why do you need to wrap it in another promise?! As I see it, you just need to do
```javascript
function getMovie(id) {
  return helper.mustBeInArray(movies, id);
}
```